Sources/Build/misc.swift

```diff
@@ -61,6 +61,16 @@ extension ClangModule {
 }
 
 extension Product {
+    /// Creates output path of a product by appending working directory and product outname.
+    ///
+    /// - Parameters:
+    ///     - prefix: Working directory in which product will be created.
+    ///
+    /// - Returns: Output path of the product.
+    func outpath(_ prefix: String) -> String {
```

Comment:
How about  `func outpath(in prefix)` or  `func outpath(with prefix)`
I feel it's more consists with new [Swift API guidelines](https://developer.apple.com/videos/play/wwdc2016/403/)